assets/helpers/flashSale.js

```diff
@@ -6,14 +6,14 @@ import type { SubscriptionProduct } from './subscriptions';
 
 function inOfferPeriod(product: SubscriptionProduct): boolean {
   // Days are 1 based, months are 0 based
-  const startTime = new Date(2018, 5, 18, 0, 0).getTime(); // 18th June 2018
-  const endTime = new Date(2018, 6, 1, 0, 0).getTime(); // 1st July 2018
+  const startTime = new Date(2018, 7, 20, 0, 0).getTime(); // 20th August 2018
+  const endTime = new Date(2018, 8, 2, 0, 0).getTime(); // 2nd September 2018
```

Comment:
Same question as on the other PR, is it correct that the sale will finish on the 1st at midnight?